src/wrappers/cryptosignwrapper.cpp

```diff
@@ -0,0 +1,46 @@
+#include "cryptosignwrapper.h"
```

Comment:
Perhaps remove / hide `getPrivateKey()`, and rename `getSeed()` to `getSecretKey()`?

CC: @AdoAdoAdo, @ccorcoveanu 

https://github.com/ElrondNetwork/elrond-sdk-erdjs/tree/main/src/walletcore

Reply:
Made it consistent now. It is called SecretKey everywhere.